Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -101,7 +104,43 @@ public GitHub3Plugin() : base(true)
 
         public override IEnumerable<ISetting> GetSettings()
         {
-            yield return OAuthToken;
+            yield return PersonalAccessToken;
+
+            var generateTokenLink = new LinkLabel { Text = _generateToken.Text };
+            generateTokenLink.Click += GenerateTokenLink_Click;
+            yield return new PseudoSetting(generateTokenLink);
+
+            var manageTokenLink = new LinkLabel { Text = _manageToken.Text };
+            manageTokenLink.Click += ManageTokenLink_Click;
+            yield return new PseudoSetting(manageTokenLink);
+        }
+
+        private void GenerateTokenLink_Click(object sender, EventArgs e)
+        {
+            OpenLink($"https://{GitHubHost.ValueOrDefault(Instance.Settings)}/settings/tokens/new?description=Token%20for%20GitExtensions&scopes=repo,public_repo");
```

Comment:
This should be using OsShellUtil.OpenUrlInDefaultBrowser

Reply:
It seems not possible because `OsShellUtil`is part of `GitUI` project that is not a dependency to the plugins.